x-pack/plugins/maps/kibana.json

```diff
@@ -25,15 +25,15 @@
     "mapsEms",
     "savedObjects",
     "share",
-    "presentationUtil",
-    "screenshotMode"
+    "presentationUtil"
   ],
   "optionalPlugins": [
     "cloud",
     "customIntegrations",
     "home",
     "savedObjectsTagging",
     "charts",
+    "screenshotMode",
```

Comment:
moving "screenshotMode" to optionalPlugins since its not required for maps to work